pkg/adapter/nimbus-kyverno/processor/kpbuilder.go

```diff
@@ -68,21 +70,131 @@ func BuildKpsFrom(logger logr.Logger, np *v1alpha1.NimbusPolicy) []kyvernov1.Pol
 }
 
 // buildKpFor builds a KyvernoPolicy based on intent ID supported by Kyverno Policy Engine.
-func buildKpFor(id string, np *v1alpha1.NimbusPolicy) ([]kyvernov1.Policy, error) {
+func buildKpFor(id string, np *v1alpha1.NimbusPolicy, logger logr.Logger) ([]kyvernov1.Policy, error) {
 	var kps []kyvernov1.Policy
 	switch id {
 	case idpool.EscapeToHost:
-		kps = append(kps, escapeToHost(np, np.Spec.NimbusRules[0].Rule))
+		kps = append(kps, escapeToHost(np))
 	case idpool.CocoWorkload:
 		kpols, err := cocoRuntimeAddition(np)
 		if err != nil {
 			return kps, err
 		}
 		kps = append(kps, kpols...)
+	case idpool.VirtualPatch:
+		kpols, err := virtualPatch(np, logger)
+		if err != nil {
+			return kps, err
+		}
+		kps = append(kps, kpols...)
+		watchCVES(np, logger)
 	}
 	return kps, nil
 }
 
+func watchCVES(np *v1alpha1.NimbusPolicy, logger logr.Logger) {
+	rule := np.Spec.NimbusRules[0].Rule
+	schedule := "0 0 * * *"
+	if rule.Params["schedule"] != nil {
+		schedule = rule.Params["schedule"][0]
+	}
+    // Schedule the deletion of the Nimbus policy
+    c := cron.New()
+    _, err := c.AddFunc(schedule, func() {
+        logger.Info("Checking for CVE updates and updation of policies")
+        err := deleteNimbusPolicy(np, logger)
+        if err != nil {
+            logger.Error(err, "error while updating policies")
+        }
+    })
+    if err != nil {
+        logger.Error(err, "error while adding the schedule to update policies")
+    }
+    c.Start()
+
+}
+
+
+
+func deleteNimbusPolicy(np *v1alpha1.NimbusPolicy, logger logr.Logger) error {
+    nimbusPolicyGVR := schema.GroupVersionResource{Group: "intent.security.nimbus.com", Version: "v1alpha1", Resource: "nimbuspolicies"}
+	err := client.Resource(nimbusPolicyGVR).Namespace(np.Namespace).Delete(context.TODO(), np.Name,metav1.DeleteOptions{})
+	if err != nil {
+		return fmt.Errorf("failed to delete Nimbus Policy: %s", err.Error())
+	}
+	logger.Info("Nimbus policy deleted successfully")
+    return nil
+}
+
+func escapeToHost(np *v1alpha1.NimbusPolicy) kyvernov1.Policy {
+	rule := np.Spec.NimbusRules[0].Rule
+	var psa_level api.Level = api.LevelBaseline
```

Comment:
Follow the `camelCase` convention for variables everywhere.
```suggestion
	var psaLevel api.Level = api.LevelBaseline
```